tools/runspeedcheck.rb

```
@@ -190,8 +197,9 @@ def gamestatistics(elapsed)
   vari = elapsed.map {|elem| (elem - avg) * (elem - avg)}.sum.fdiv(dof);
   sd = Math.sqrt(vari)
   min, max = elapsed.minmax
+  median = elapsed[n / 2];
 
-  puts "#{'%.3f' % avg} ± #{'%.3f' % sd} sec (#{'%.3f' % min} -- #{'%.3f' % max}) N:#{n}"
+  puts "#{'%.3f' % avg} ± #{'%.3f' % sd} sec (#{'%.3f' % min} -- #{'%.3f' % median} -- #{'%.3f' % max}) N:#{n}"
 end
```

RuboCop (1.82.1)

[convention] Assignment Branch Condition size for `gamestatistics` is too high. [<9, 21, 3> 23.04/23]

(Metrics/AbcSize)